docs/supported-languages.md

```diff
@@ -4,30 +4,92 @@ id: supported-languages
+import SscIntro from "/src/components/concept/_ssc-intro.md"
 import MoreHelp from "/src/components/MoreHelp"
 
+<ul id="tag__badge-list">
+{
+Object.entries(frontMatter).filter(
+    frontmatter => frontmatter[0] === 'tags')[0].pop().map(
+    (value) => <li class='tag__badge-item'>{value}</li> )
+}
+</ul>
+
 # Supported languages
 
-## Language maturity
+This document provides information about supported languages and language maturity definitions for the following products:
+
+* Semgrep
+* Semgrep Supply Chain
+
+## Semgrep
+
+Semgrep is a fast, open source, static analysis engine for finding bugs and enforcing code standards.
+
+### Language maturity
 
 <SupportedLanguagesTable />
 
-## Maturity definitions
+### Maturity definitions
 
+#### Language maturity factors
 Language maturity is determined by 3 factors in the Semgrep ecosystem:
 
-1. **Parse rate** - how well Semgrep can parse code in a given language.
-1. **Feature support** - what [Semgrep features](writing-rules/pattern-syntax.mdx) are implemented for a given language.
-1. **Ruleset count** - number of [Semgrep rule groupings](https://semgrep.dev/explore) in the cloud app.
+<dl>
+    <dt>Parse rate</dt>
+    <dd>How well Semgrep can parse code in a given language.</dd>
+    <dt>Feature support</dt>
+    <dd>What <a href='/writing-rules/pattern-syntax/'>Semgrep features</a> are implemented for a given language.</dd>
+    <dt>Ruleset count</dt>
+    <dd>Number of <a href='https://semgrep.dev/explore/'>Semgrep rule groupings</a> in Semgrep Registry.</dd>
+</dl>
+
+#### Levels of maturity
 
-There are 3 levels of maturity: **experimental**, **beta**, and **generally available (GA)**. Each of these maturity levels are combined with a threshold of the factors above. When a language meets the maturity threshold for each of the factors then it’s moved into that maturity level.
+There are 3 levels of maturity: 
 
-Generally speaking, the features-by-maturity level are roughly: **experimental)** syntax support, ellipsis operator support, and basic metavariable support, **beta)** everything prior and nearly complete metavariable support and metavariable equality, **GA)** everything prior and all advanced features like regexp, equivalence, deep expression operator, typing, etc.
+<dl>
+<dt>Experimental</dt>
+<dd>Experimental languages support the following:
+<ul>
+    <li>Syntax</li>
+    <li>Ellipsis operator</li>
+    <li>Basic metavariable functionality</li>
+</ul>
+</dd>
+<dt>Beta</dt>
+<dd>Beta languages support the following:
+<ul>
+    <li>All features supported in Experimental</li>
+    <li>Complete metavariable support</li>
+    <li>Metavariable equality</li>
+</ul>
+</dd>
+<dt>Generally available</dt>
+<dd>Generally available languages support all advanced features such as the following:
+<ul>
+<li>All features supported in Beta</li>
+<li>Regexp</li>
+<li>Equivalence</li>
+<li>Deep expression operator</li>
+<li>Types and typing</li>
+</ul>
+</dd>
+</dl>
```

Comment:
This is great. Thank you. Three questions:

1. Should we remove the support level information given under our modularized table for Semgrep support?
    See below:
    ![Screenshot 2022-11-07 at 13 41 20](https://user-images.githubusercontent.com/26669307/200312973-8bce5f07-6b4c-4a56-96c0-33dde7624443.png)

2. I am also not sure about the use of emojis in our table. Do they look ok? I mean... they are definitely cheerful. Not sure if they look completely professional. :D (We don't need to make everything dull though.)
    ![Screenshot 2022-11-07 at 13 41 37](https://user-images.githubusercontent.com/26669307/200313025-ef84819a-7f22-4837-9d25-2da07cbc3264.png)

3. Should I also rewrite the language support table to HTML as in the tables below? The table headers have a central alignment etc. so maybe we can unify the style. If you say yes, can I push it as a commit into this branch? 🙏

Reply:
I think the emoji is appropriate for the table.

Reply:
@s-santillan What about points 1. and 3.

Reply:
Maybe we can remove it and just link to the # Levels of maturity section (by an absolute link from the module).

Reply:
Support level != features btw so we should probably keep it ("expect bugs at this level, we won't prioritize features at this level", etc.)